crates/router/src/core/payments/routing.rs

```
@@ -466,7 +468,26 @@ pub async fn perform_static_routing_v1(
                 }
             };
 
-            execute_dsl_and_get_connector_v1(backend_input, interpreter)?
+            let de_euclid_connectors = perform_decision_euclid_routing(
+                state,
+                backend_input.clone(),
+                business_profile.get_id().get_string_repr().to_string(),
+            )
+            .await
+            .map_err(|e|
+                logger::error!(decision_engine_euclid_evaluate_error=?e, "decision_engine_euclid: error in evaluation of rule")
+            ).unwrap_or_default();
```

Add a comment on why we are ignoring errors in this case